### PR TITLE
Layout algorithm

### PR DESCRIPTION
**Pull request by BugOrFeature**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (Jipje) [inline .idea/vcs.xml]: .idea should be in the gitignore....

File: src/main/java/graph/SequenceGraph.java [hunk 2]
```diff
@@ -1,133 +1,274 @@
...
+        this.initialized = true;
+    }
+
+    /** Creates a topological ordering of the graph and returns this ordering in the form of a LinkedList
+     *
+     * @return list - LinkedList<Integer>
+     */
+    private LinkedList<Integer> createTopologicalOrder() {
+        LinkedList<Integer> queue = new LinkedList<Integer>();
+        LinkedList<Integer> list = new LinkedList<Integer>();
```
Comment (Jipje) [inline src/main/java/graph/SequenceGraph.java]: Are these classes tested?

File: src/main/java/graph/SequenceGraph.java [hunk 3]
```diff
@@ -1,133 +1,274 @@
...
-    private void createColumns() {
-        for (int i = 1; i <= size; i++) {
-            SequenceNode parent = nodes.get(i);     // Start at first node
-            ArrayList<SequenceNode> children = parent.getChildren();    // Get all children
-            for (SequenceNode child : children) {
-                child.incrementColumn(parent.getColumn());      // Assign layer
+    private void createLayers() {
+        LinkedList<Integer> list = getDeepCopyTopologicalOrderedList();
+
+        AbstractNode firstNode = this.getNode(list.peek());
```
Comment (Jipje) [inline src/main/java/graph/SequenceGraph.java]: Looks good!

File: src/main/java/graph/SequenceNode.java [hunk 4]
```diff
@@ -1,31 +1,136 @@
...
+    private int xCoordinate;
+    private int yCoordinate;
 
-    public SequenceNode(int id) {
+
+    public SequenceNode(int id, String seq){
         this.id = id;
+        this.seq = seq;
+        this.adjacencyList = new ArrayList<Integer>();
+        this.parents = new ArrayList<Integer>();
```
Comment (Jipje) [inline src/main/java/graph/SequenceNode.java]: Maybe this is stiil too much information for in a node. But that is a big change we can do later. Looks good anyway.

**Comment by BugOrFeature** [comment]
changed the implementation with another pull request. This one can be closed for now.

**Pull request closed without merging**